### PR TITLE
Replace axios with Node provided modules

### PR DESCRIPTION
**Pull request by Achal1607**
This PR aims to remove the axios module and instead use 'https' package provided by Node itself.

### CODE REVIEW

File: vscode/src/jdkDownloader.ts [hunk 1]
```diff
@@ -29,6 +29,22 @@ import { promisify } from 'util';
 let customView: vscode.WebviewPanel;
 let logger: vscode.OutputChannel;
 
+export const getSha256Checksum = async (url: string) => {
+  return new Promise((resolve, reject) => {
+      https.get(`${url}.sha256`, res => {
+          let data = '';
```
Comment (sid-srini) [inline vscode/src/jdkDownloader.ts:35]: Perhaps you may add the check for the response status code 200 before attempting to read the data. Similar to the actual file download itself in the function `JDKDownloader`.

**Review by sid-srini — CHANGES_REQUESTED** [review]
This PR would be very useful in removing an extra third-party library dependency. 
Please do remove the `axios` imports from *jdkDownloader.ts* and the dependency from *package.json* and *package-lock.json*. Thanks.

File: vscode/src/jdkDownloader.ts [hunk 2]
```diff
@@ -28,9 +27,28 @@ import { promisify } from 'util';
 
 let customView: vscode.WebviewPanel;
 let logger: vscode.OutputChannel;
+const ALGORITHM = 'sha256';
+
+export const getSha256Checksum = async (url: string) => {
+  return new Promise((resolve, reject) => {
+      https.get(`${url}.${ALGORITHM}`, res => {
+          if (res.statusCode !== 200) {
+            return `HTTP Error ${res.statusCode} - ${res.statusMessage}`;
```
Comment (sid-srini) [inline vscode/src/jdkDownloader.ts:36]: Should it throw an error here with the message string being returned here?
That way the rejection handler could appropriately notify the user.

**Comment by Achal1607** [comment]
Closing since changes done in #248

**Pull request closed without merging**